hack/test-integration.sh

```diff
@@ -37,21 +37,25 @@ mkdir -p "${testdir}"
 
 # build the test executable (cgo must be disabled to have the symbol table available)
 pushd "${testdir}" 2>&1 >/dev/null
-CGO_ENABLED=0 go test -c -tags="${tags}" "${OS_GO_PACKAGE}/${package}"
+CGO_ENABLED=0 go test -v -c -tags="${tags}" "${OS_GO_PACKAGE}/${package}"
 popd 2>&1 >/dev/null
 
 start_etcd
 trap cleanup EXIT SIGINT
 
 function exectest() {
-  out=$("${testexec}" -test.run="^$1$" "${@:2}" 2>&1)
-  res=$?
-  if [[ ${res} -eq 0 ]]; then
-    echo ok $1
-    exit 0
+  if [[ ! -z "${VERBOSE-}" ]]; then
+    ${testexec} -logtostderr=true -test.v=true -v=5 -test.run="^$1$" "${@:2}"
```

Comment:
Can't we use `VERBOSE` to set actual verbosity log instead of hardcoding 5 here.